codecs/opus/opus-native/macosx/pom.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <packaging>dylib</packaging>
+
+    <parent>
+        <groupId>org.restcomm.media.codecs.opus</groupId>
+        <artifactId>opus-native</artifactId>
+        <version>6.0.0-SNAPSHOT</version>
+    </parent>
+
+    <groupId>org.restcomm.media.codecs.opus</groupId>
+    <artifactId>libopus_jni</artifactId>
+    <name>Opus Native Mac OSX</name>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>native-maven-plugin</artifactId>
+                <extensions>true</extensions>
+
+                <configuration>
+                    <compilerProvider>generic</compilerProvider>
+                    <compilerStartOptions>
+                        <compilerStartOption>-std=c++11 -I${java.home}/../include -I${java.home}/../include/linux -I/usr/include/opus -Wall -fPIC</compilerStartOption>
```

Comment:
`-I${java.home}/../include/linux` does not exist on macos. Maybe `-I${java.home}/../include/darwin`?